docs/develop-logseq-on-windows.md

```diff
@@ -9,6 +9,7 @@ This is a guide on creating Logseq development environment on Windows with `Powe
 * Node.js 16.x
 * Clojure (follow this [Guidance](https://clojure.org/guides/getting_started#_installation_on_windows))
 * JRE 8 (required for Clojure)
+* Visual Studio (required for desktop app)
```

Comment:
Actually, Visual Studio Build Tools 2019/2022 is enough. The required components are:
- MSVC 
- Windows 10 SDK